src/components/SearchBar/SearchBar.styles.ts

```diff
@@ -0,0 +1,128 @@
+import { Box, TextField, IconButton } from '@mui/material';
+import { styled } from '@mui/material/styles';
+
+export const SearchContainer = styled(Box)(({ theme }) => ({
+  position: 'absolute',
+  width: '324px',
+  height: '67px',
+  left: '929px',
+  top: '91px',
+  background: 'rgba(250, 250, 250, 0.75)', // Fondo con opacidad
+  backdropFilter: 'blur(10px)', // Añadir blur para efecto vidrio
+  borderRadius: '15px',
+  display: 'flex',
+  alignItems: 'center',
+  boxSizing: 'border-box',
+  border: '1px solid rgba(255, 255, 255, 0.2)', // Borde sutil
```

Comment:
Border is set using a hardcoded rgba string. Use theme.palette/divider or alpha(theme.palette.common.white, x) to keep colors consistent with the theme.

**Sugerencia:**
```typescript
Use the theme token, e.g. border: `1px solid ${alpha(theme.palette.divider, 0.2)}` or borderColor: theme.palette.divider.
```